### ISSUE TITLE
a paper of the source code

### ISSUE THREAD

**Issue by LJtang6737** [rajibhossen/edge-offloading#1]
Hello, author. can you provide a paper of the source code ? Thank you very much. I will continue to pay attention.

**Comment by rajibhossen** [comment]
Here it is - https://dl.acm.org/doi/abs/10.1145/3466826.3466838

**Comment by LJtang6737** [comment]
very glad to hear from you. When I run the dqn_lstm.py file, get the dqn_lstm model, and then test it on the test data, I find that the offloading decisions are either all executed locally or all offloaded to the edge server, is this normal?

**Comment by rajibhossen** [comment]
No. the model should offload based on task features such as job size, available energy, etc.

**Comment by LJtang6737** [comment]
I run the dqn_lstm.py file, get the dqn_lstm model. And then ,I made several attempts, running the dqn_lstm model with the test data, choosing the action with the largest Q value, without exploring, the offloading decisions are still either all executed locally or all offloaded to the edge server. Do you have any good suggestions or know where the problem might be?

**Comment by sidj9900** [comment]
Facing same issue
